tasks/restore_vm.yml

```diff
@@ -10,16 +10,19 @@
     environment: "{{ cohesity_restore_vm.environment | default('Physical') }}"
     job_name: "{{ cohesity_restore_vm.job_name | default('') }}"
     endpoint: "{{ cohesity_restore_vm.endpoint | default('') }}"
+    restore_to_source: "{{ cohesity_restore_vm.restore_to_source | default('') }}"
```

Comment:
instead of boolean this should be the actual vcenter name . It'll lead to confusion if its a boolean. And default if not specified should be the original vcenter.

Reply:
this is not a boolean, its a string which takes vCenter. By default it restores to original location

Reply:
Cool ok . LGTM